src/async/stompest/async/client.py

```diff
@@ -135,14 +135,16 @@ def session(self):
     # STOMP commands
     #
     @defer.inlineCallbacks
-    def connect(self, headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None):
-        """connect(headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None)
+    def connect(self, headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None, certKey=None, privateKey=None):
+        """connect(headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None, certKey=None, privateKey=None)
```

Comment:
Yes, I see now that the certfile and keyfile are not available on the context after the load call.  This might be because those parameters are allowed to be either file names _or file-like_.

Another idea might be to store the certfile and keyfile as additional attributes on the `config` object, and then make the `config` object available in the place where `clientFromString()` is called.  I _feel like_ there is a right solution here, but it probably involves some additional Twisted knowledge.  I'll defer to @nikipore here.  Clearly, it's the Twisted stuff that is doing extra things with the handling of the SSL connection, and if we must go via `clientFromString()` then we'll have to provide the certfile and keyfile into that command, as you've already done.